backend/wmg/pipeline/summary_cubes/cell_count.py

```diff
@@ -63,6 +65,13 @@ def create_cell_count_cube(corpus_path: str):
     """
     obs = extract(corpus_path)
     df = transform(obs)
+
+    cell_types = list(df["cell_type_ontology_term_id"])
+    n_cells = np.array(list(df["n_cells"]))
+    (n_cells_agg,) = aggregate_across_cell_type_descendants(cell_types, [n_cells])
+    df["n_cells"] = n_cells_agg
```

Comment:
aggregate counts are default, raw counts are now `n_cells_raw`

Reply:
Might be worth noting that as a comment somewhere. Raw is a bit of an overloaded term